sdk/identity/azure-identity/azure/identity/aio/_credentials/default.py

```diff
@@ -89,14 +89,4 @@ def __init__(self, **kwargs):
         :raises ~azure.core.exceptions.ClientAuthenticationError: authentication failed. The exception has a
           `message` attribute listing each authentication attempt and its error message.
         """
-        try:
-            return await super(DefaultAzureCredential, self).get_token(*scopes, **kwargs)
-        except ClientAuthenticationError as e:
-            raise ClientAuthenticationError(
-                message="""
-{}\n\nPlease visit the documentation at
-https://aka.ms/python-sdk-identity#defaultazurecredential
-to learn what options DefaultAzureCredential supports""".format(
-                    e.message
-                )
-            )
+        return await super(DefaultAzureCredential, self).get_token(*scopes, **kwargs)
```

Comment:
```suggestion
        return await super().get_token(*scopes, **kwargs)  # pylint:disable=useless-super-delegation
```

Reply:
We can use the magic `super()` because we're definitely running on Python 3 here:
```suggestion
        return await super().get_token(*scopes, **kwargs)
```